security/fma-shared-lockbox.md

```diff
@@ -0,0 +1,136 @@
+# Shared Lockbox: Failure Modes and Recovery Path Analysis
+
+| Author | Joxes, Agusduha, Gotzen |
+| --- | --- |
+| Created at | 2024-12-20 |
+| Initial Reviewers | Pending |
+| Needs Approval From | Pending |
```

Comment:
This is the only one that needs an actual security reviewer, I'll let you know who to put here